NBitcoin.TestFramework/NodeBuilder.cs

```diff
@@ -19,677 +19,677 @@
-				return new IPEndPoint(IPAddress.Parse("127.0.0.1"), ports[0]);
-			}
-		}
-
-		public string Config
-		{
-			get
-			{
-				return _Config;
-			}
-		}
-
-		private readonly NodeConfigParameters _ConfigParameters = new NodeConfigParameters();
-		private string _Config;
-
-		public NodeConfigParameters ConfigParameters
-		{
-			get
-			{
-				return _ConfigParameters;
-			}
-		}
-
-		public CoreNode(string folder, NodeBuilder builder)
-		{
-			this._Builder = builder;
-			this._Folder = folder;
-			_State = CoreNodeState.Stopped;
-
-			dataDir = Path.Combine(folder, "data");
-			var pass = Hashes.Hash256(Encoding.UTF8.GetBytes(folder)).ToString();
-			creds = new NetworkCredential(pass, pass);
-			_Config = Path.Combine(dataDir, "bitcoin.conf");
-			ConfigParameters.Import(builder.ConfigParameters, true);
-			ports = new int[2];
-
-			if(builder.CleanBeforeStartingNode && File.Exists(_Config))
-			{
-				var oldCreds = ExtractCreds(File.ReadAllText(_Config));
-				CookieAuth = oldCreds == null;
-				ExtractPorts(ports, File.ReadAllText(_Config));
-
-				try
-				{
-
-					this.CreateRPCClient().Stop();
-				}
-				catch
-				{
-					try
-					{
-						CleanFolder();
-					}
-					catch
-					{
-						throw new InvalidOperationException("A running instance of bitcoind of a previous run prevent this test from starting. Please close bitcoind process manually and restart the test.");
-					}
-				}
-				CancellationTokenSource cts = new CancellationTokenSource();
-				cts.CancelAfter(10000);
-				while(!cts.IsCancellationRequested && Directory.Exists(_Folder))
-				{
-					try
-					{
-						CleanFolder();
-						break;
-					}
-					catch { }
-					Thread.Sleep(100);
-				}
-				if(cts.IsCancellationRequested)
-					throw new InvalidOperationException("You seem to have a running node from a previous test, please kill the process and restart the test.");
-			}
-
-			CookieAuth = NodeImplementation.SupportCookieFile;
-			Directory.CreateDirectory(folder);
-			Directory.CreateDirectory(dataDir);
-			FindPorts(ports);
-		}
-
-		public string GetRPCAuth()
-		{
-			if(!CookieAuth)
-				return creds.UserName + ":" + creds.Password;
-			else
-				return "cookiefile=" + Path.Combine(dataDir, this._Builder.NodeImplementation.RegtestFolderName ?? "regtest", ".cookie");
-		}
-
-		private void ExtractPorts(int[] ports, string config)
-		{
-			var p = Regex.Match(config, "rpcport=(.*)");
-			ports[1] = int.Parse(p.Groups[1].Value.Trim());
-		}
-
-		private NetworkCredential ExtractCreds(string config)
-		{
-			var user = Regex.Match(config, "rpcuser=(.*)");
-			if(!user.Success)
-				return null;
-			var pass = Regex.Match(config, "rpcpassword=(.*)");
-			return new NetworkCredential(user.Groups[1].Value.Trim(), pass.Groups[1].Value.Trim());
-		}
-
-		public Network Network
-		{
-			get; set;
-		} = Network.RegTest;
-
-		private void CleanFolder()
-		{
-			try
-			{
-				Directory.Delete(_Folder, true);
-			}
-			catch(DirectoryNotFoundException) { }
-		}
+{

```

Comment:
should be tabs

Reply:
It was supposed to be tabs. Anyway I fixed that.